src/stories/containers/TransparencyReport/utils/actualsTableHelpers.tsx

```diff
@@ -40,9 +54,11 @@ export const getActualsBreakdownItems = (
 
     for (const groupedCatKey in groupedCategory) {
       if (
-        Math.abs(getGroupForecast(groupedCategory[groupedCatKey], month)) +
-          Math.abs(getGroupActual(groupedCategory[groupedCatKey], month)) ===
-        0
+        Math.abs(getGroupForecast(groupedCategory[groupedCatKey], month)) === 0 &&
+        Math.abs(getGroupActual(groupedCategory[groupedCatKey], month)) === 0 &&
+        Math.abs(getGroupMonthlyBudget(grouped[groupedKey], month)) === 0 &&
+        Math.abs(getGroupDifference(grouped[groupedKey], month)) === 0 &&
+        Math.abs(getGroupPayment(grouped[groupedKey], month)) === 0
```

Comment:
`Math.abs` is not needed to compare with `0`

Reply:
you are right, done